{
	"schema_version": "1.6.0",
	"id": "RHSA-2020:3756",
	"modified": "2024-09-16T04:42:37Z",
	"published": "2024-09-16T04:42:37Z",
	"related": [
		"CVE-2020-14352"
	],
	"summary": "Red Hat Security Advisory: librepo security update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:H/I:H/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:rhel_e4s:8.0::baseos",
				"name": "librepo",
				"purl": "pkg:rpm/redhat/librepo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:1.9.2-2.el8_0"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:3756.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_e4s:8.0::baseos",
				"name": "librepo-debuginfo",
				"purl": "pkg:rpm/redhat/librepo-debuginfo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:1.9.2-2.el8_0"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:3756.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_e4s:8.0::baseos",
				"name": "librepo-debugsource",
				"purl": "pkg:rpm/redhat/librepo-debugsource"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:1.9.2-2.el8_0"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:3756.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_e4s:8.0::baseos",
				"name": "python3-librepo",
				"purl": "pkg:rpm/redhat/python3-librepo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:1.9.2-2.el8_0"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:3756.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_e4s:8.0::baseos",
				"name": "python3-librepo-debuginfo",
				"purl": "pkg:rpm/redhat/python3-librepo-debuginfo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:1.9.2-2.el8_0"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2020:3756.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2020:3756"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#important"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1866498"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2020/rhsa-2020_3756.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2020-14352"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2020-14352"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-14352"
		}
	]
}
